ov/hypervisor_cluster_profile.go

```diff
@@ -352,6 +353,70 @@ func (c *OVClient) DeleteHypervisorClusterProfile(name string) error {
 	return nil
 }
 
+func (c *OVClient) DeleteHypervisorClusterProfileSoftDelete(name string, soft_delete bool) error {
+	err_softdelete := c.DeleteHypervisorClusterProfileSoftDeleteForce(name, soft_delete, false)
+	return err_softdelete
+}
+
+func (c *OVClient) DeleteHypervisorClusterProfileSoftDeleteForce(name string, soft_delete bool, force bool) error {
```

Comment:
Please try to create a abstract function that works for both when softDelete is true or false